samples/wifi/promiscuous/src/main.c

```diff
@@ -107,6 +111,33 @@ K_THREAD_DEFINE(receiver_thread_id, STACK_SIZE,
 #define PKT_FC_SUBTYPE_MASK 0xF0
 #define PKT_FC_SUBTYPE_SHIFT 4
 
+#ifdef CONFIG_NET_CAPTURE
+static const struct device *capture_dev;
+#endif
+
+#ifdef CONFIG_USB_DEVICE_STACK
+#include <zephyr/usb/usb_device.h>
+
+/* Fixed address as the static IP given from Kconfig will be
+ * applied to Wi-Fi interface.
+ */
+#define CONFIG_NET_CONFIG_USB_IPV4_ADDR "192.0.2.1"
```

Comment:
See https://github.com/nrfconnect/sdk-nrf/commit/a2e4806e96d2fa1188d7cce6413a2f526b13bb52 if you have no plans to make this configurable.